bulk_insert_test.go

```diff
@@ -113,6 +113,59 @@ func Test_insertObject(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func Test_insertObjSetWithCallback(t *testing.T) {
+	type Table struct {
+		ID            uint `gorm:"primary_key;auto_increment"`
+		RegularColumn string
+		Custom        string `gorm:"column:ThisIsCamelCase"`
+	}
+
+	db, mock, err := sqlmock.New()
+	require.NoError(t, err)
+
+	defer db.Close()
+
+	gdb, err := gorm.Open("mysql", db)
```

Comment:
The feature you're testing is PostgreSQL specific so it would make more sense to use the postgres dialect here even though it's just a mock.